packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.html

```diff
@@ -4,19 +4,26 @@
       <slot></slot>
       <div
         class="tab-bar_more-button slds-tabs_default__item slds-tabs_default__overflow-button slds-is-absolute slds-p-right_none slds-p-left_none"
-        title={labels.moreTabs} role="presentation">
+        title={labels.moreTabs}
+        role="presentation"
+      >
         <div class={dropdownClasses}>
-          <button class="slds-button slds-var-p-horizontal_small" aria-haspopup="true"
-            onclick={toggleDropdown}>{moreButtonLabel}
-            <lightning-icon class={moreButtonIconClasses} icon-name={arrowIconName} size="xx-small" aria-hidden="true">
-            </lightning-icon>
+          <button class="slds-button slds-var-p-horizontal_small" aria-haspopup="true" onclick={toggleDropdown}>
+            {moreButtonLabel}
+            <lightning-icon class={moreButtonIconClasses} icon-name={arrowIconName} size="xx-small" aria-hidden="true"> </lightning-icon>
           </button>
           <div class="slds-dropdown slds-dropdown_right">
             <ul class="slds-dropdown__list slds-dropdown_length-with-icon-10" role="menu">
               <template for:each={tabsInDropdown} for:item="tab">
                 <li class="slds-dropdown__item" role="presentation" key={tab.id}>
-                  <a role="menuitem" tabindex={optionTabIndex} onclick={handleDropdownTabSelect} data-value={tab.value}
-                    data-label={tab.label}>
+                  <a
+                    role="menuitem"
+                    tabindex={optionTabIndex}
+                    onclick={handleDropdownTabSelect}
+                    onkeydown={onKeyDown}
```

Comment:
I think we can add the tab status here too :)

Reply:
How do you define tab status? The tabs right now uses `updateDropdownOptions` to get the "tabs" that are overflow and put them in dropdown menu. Once the tab in dropdown menu is selected, it would be removed from dropdown menu and another tab would be added to the dropdown menu to make sure the tab section is no overflowing. Under this setup, it seems all tabs in dropdown menu could never have the state "selected"

Reply:
added aria-pressed for values under dropdown menu